server/src/main/scala/org/http4s/server/websocket/package.scala

```diff
@@ -4,15 +4,14 @@ package server
 import org.http4s.websocket.Websocket
 import org.http4s.websocket.WebsocketBits.WebSocketFrame
 
-import scalaz.stream.{Process, Sink}
+import scalaz.stream.{Exchange, Process, Sink}
 import scalaz.concurrent.Task
 import Process._
 
 package object websocket {
   val websocketKey = AttributeKey.http4s[Websocket]("websocket")
 
-  def WS(source: Process[Task, WebSocketFrame] = halt,
-         sink: Sink[Task, WebSocketFrame] = halt,
+  def WS(exchange: Exchange[WebSocketFrame, WebSocketFrame],
```

Comment:
I wonder if there should remain a convenience overoad to create the exchange from the source and sink?  I'm trying to anticipate how often we'll end up calling just `WS(Exchange(...))`.